docs/usage/beacon-management.md

```diff
@@ -33,7 +33,12 @@ You must generate a secret 32-byte (64 characters) hexadecimal string that will
 
 ### Configure Lodestar to locate the JWT secret
 
-When starting up a Lodestar beacon node in any configuration, ensure you add the `--jwt-secret $JWT_SECRET_PATH` flag to point to the saved secret key file.
+When starting up a Lodestar beacon node in any configuration, ensure you add the `--jwtSecret $JWT_SECRET_PATH` flag to point to the saved secret key file.
+
+### Set up and include identifiers in JWT tokens
+
+Lodestar auto-populates `clv` field in the claims of JWT authentication tokens with a non-configurable value `Lodestar/$CLIENT_VERSION` eg. `Lodestar/v1.3.0/2d0938e` to communicate the client's version. Lodestar also optionally includes `id` field in the claims with value `$JWT_ID` if the appropriate flag `--jwtId $JWT_ID` is added. 
+`id` and `clv` are particularly useful when running multiple consensus-layer clients with the different JWT secrets which makes the execution-layer client difficult to choose which JWT secret to verify against due to the inability to distinguish between the different consensus-layer clients. 
```

Comment:
Just a side note, I'd rather see use the terms "consensus client" and "execution client" in the docs. This seems to be the more conventional terminology and it's easier to read / less verbose

Reply:
> Just a side note, I'd rather see use the terms "consensus client" and "execution client" in the docs. This seems to be the more conventional terminology and it's easier to read / less verbose

We can have a separate PR to systemically update the term "consensus-layer client" and "execution-layer client"